doc/user_guide/user_guide.md

```diff
@@ -56,8 +56,12 @@ models on the Exasol Cluster. More information on The BucketFS can be found
 - The Exasol cluster must already be running with version 7.1 or later.
 - DB connection information and credentials are needed.
 
-### BucketFS Connection 
-An Exasol connection object must be created with Exasol BucketFS connection information and credentials. 
+### BucketFS Connection
+An Exasol connection object must be created with Exasol BucketFS connection information and credentials.
+Normally, the connection object is created as part of the Transformers Extension deployment 
+(see the [Setup section](#deploy-the-extension-to-the-database) below). This section describes how this object
+can be created manually.
```

Comment:
To be picky: each heading should be followed by an empty line.

Additionally, in the past we found out, that paragraphs split into multiple lines can small updates (e.g. inserting a single word) to potentially cause a diff affecting multiple lines. Hence, we usually prefer to format paragraphs in a single source line only. 

I personally must admit, that multiple lines are much more pleasant to read in the editor :slightly_frowning_face:

Reply:
"each heading should be followed by an empty line." I can't see a difference in the viewer on my screen.